### PR TITLE
FizzBuzz- groovy

### PR DESCRIPTION
**Pull request by sukeshni**
@shunjikonishi Please help
- Build Successful 
- No test is running for groovy 

```
$ codecheck
codecheck version 0.4.3
:compileJava UP-TO-DATE
:compileGroovy UP-TO-DATE
:processResources UP-TO-DATE
:classes UP-TO-DATE
:compileTestJava UP-TO-DATE
:compileTestGroovy UP-TO-DATE
:processTestResources UP-TO-DATE
:testClasses UP-TO-DATE
:test UP-TO-DATE

BUILD SUCCESSFUL

Total time: 3.113 secs

This build could be faster, please consider using the Gradle Daemon: https://docs.gradle.org/2.8/userguide/gradle_daemon.html
codecheck: Finish with code 0
codecheck: tests  : 0
codecheck: success: 0
codecheck: failure: 0
```
- Same problem for https://github.com/code-check-100/line-challenge-B3/tree/groovy

### CODE REVIEW

File: src/main/groovy/FizzBuzz.groovy [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+def fizzBuzz(n) {
+  //ここに素数を一文字づつ数列化した際のn番目の数値を出力するプログラムを実装してください。
+  //BEGIN_CHALLENGE
+
+  result = ""
```
Comment (shunjikonishi) [inline src/main/groovy/FizzBuzz.groovy]: I think this code is confusing.

This result var is not necessary.
The reason why this fizzBuzz function returns correct value is combination of following.
- `result = "hoge"` returns "hoge"
- if statement returns value
- `return` is able to omit

**Comment by shunjikonishi** [comment]
@sukeshni fix me

Test runs correctly.
Gradle caches code status.
So if you don't modify any main/test code, test is skipped.

If you want to run test again, you have to run `gradle clean`.

**Comment by sukeshni** [comment]
@shunjikonishi :+1: Thank you for helping out. Please review.

**Comment by shunjikonishi** [comment]
@sukeshni fix me
Still redundant `result` are present.
You can remove completely.

```
if (n % 15 == 0) "FizzBuzz"
else if (n % 5 == 0) "Buzz"
else if (n % 3 == 0) "Fizz" 
```

**Comment by sukeshni** [comment]
@shunjikonishi Removed var `result` completely. Please review.

**Comment by shunjikonishi** [comment]
LGTM

**Pull request closed without merging**